vm/entrypoint/src/syscalls/host.rs

```diff
@@ -86,3 +86,28 @@ pub extern "C" fn get_balance(value: *mut u32) {
   #[cfg(not(target_os = "zkvm"))]
   unreachable!()
 }
+
+/// Spawn a new instance of a template.
+///
+/// No return value. It either succeeds or reverts.
+/// The host calculates the new program address based on the template,
+/// the state blob, and the principal nonce. The template and nonce are
+/// available in its context and don't need to be passed here. The
+/// owner of the new program is implicit in the blob, which is just a
+/// serialized version of the instantiated template (struct) state.
+#[allow(unused_variables)]
```

Comment:
NIT: instead of disabling linter, you could provide a separate definition for the `not(zkvm)` target.